pkg/controller/datavolume/controller-base.go

```diff
@@ -365,9 +365,9 @@ func (r *ReconcilerBase) getPVC(dv *cdiv1.DataVolume) (*corev1.PersistentVolumeC
 	return pvc, nil
 }
 
-func (r *ReconcilerBase) getDataVolume(key types.NamespacedName) (*cdiv1.DataVolume, error) {
+func getDataVolume(c client.Client, key types.NamespacedName) (*cdiv1.DataVolume, error) {
```

Comment:
Hmm not sure why this one needed to be changed?

Reply:
I use this function in 2 different places in the code and one of them doesn't have access to the function this way, so I changed the signature to be a regular function and requested the client as a parameter instead.

Reply:
AFAIK you only use `func (r *SmartCloneReconciler) getDataVolume` and not this one

Reply:
Thanks Alex, I tried to fix what you mentioned.
Plz let me know if there is any problem and I'll fix it tomorrow.

Reply:
Looks good, I just think we should test this in functional tests since we are now committing that CDI will always pass labels